vhpidirect/arrays/intvector/csized/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+set -e
+
+cd $(dirname "$0")
+
+
+echo "> Analyze tb.vhd"
+ghdl -a tb.vhd
+
+
+echo "> Build tb (with caux.c) [GHDL]"
+ghdl -e -Wl,caux.c tb
+
+echo "> Execute tb"
```

Comment:
Shall I update the style guide? I did feel like the echos got a little lost, needed emboldening

Reply:
I think it would be good to use something other than plain shell scripts, which allows better management of the examples (colouring, number of passes/failures, parallel/isolated execution). These are both examples and a regression test suite. However, at the same time, we need to generate/provide simple sequences of commands, for new users to understand easily.

So, for now, as you wish. I think it is more important to focus on VHDL and a formatter for C.